src/Microsoft.Azure.WebJobs/AutoResolveAttribute.cs

```diff
@@ -25,5 +26,10 @@ public AutoResolveAttribute()
         /// in the Microsoft.Azure.WebJobs.Host assembly.
         /// </summary>
         public Type ResolutionPolicyType { get; set; }
+
+        /// <summary>
+        /// If this is empty, then specify a default. 
+        /// </summary>
+        public DefaultResolveValue Default { get; set; } = DefaultResolveValue.None;
```

Comment:
I'm not seeing any extensions that have meaningful literal default values, so I'm hesitant to add that. 
I'd recommend that extension authors just check for null and handle appropriately. 


---
In reply to: [113586086](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r113586086) [](ancestors = 113586086)